src/it/parallel-modules/.mvn/maven.config

```diff
@@ -0,0 +1 @@
+-T10
```

Comment:
is maven-invoker-plugin picking up this file automatically? I wasn't aware of this feature.